test/functional/page_objects/discover_page.ts

```diff
@@ -164,6 +164,7 @@ export class DiscoverPageObject extends FtrService {
 
   public async clickNewSearchButton() {
     await this.testSubjects.click('discoverNewButton');
+    await this.testSubjects.moveMouseTo('unifiedFieldListSidebar__toggle-collapse'); // cancel tooltips
```

Comment:
Just curious, were we running into flakiness due to this?

Reply:
Yes, the new icon buttons trigger tooltips which might cover the time range filter.